exec/java-exec/src/main/java/org/apache/drill/exec/store/log/LogFormatPlugin.java

```diff
@@ -47,23 +47,30 @@
 import org.slf4j.LoggerFactory;
 
 public class LogFormatPlugin extends EasyFormatPlugin<LogFormatConfig> {
-  public static final String PLUGIN_NAME = "logRegex";
   private static final Logger logger = LoggerFactory.getLogger(LogFormatPlugin.class);
 
+  public static final String PLUGIN_NAME = "logRegex";
+  public static final String PROP_PREFIX = TupleMetadata.DRILL_PROP_PREFIX + "regex.";
```

Comment:
Since properties are `log` specific should we add `log` in the properties naming as well as we did for `text` properties?